cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -739,3 +826,72 @@ def create_subnet(self, cidr_block, name=None):
 
     def refresh(self):
         return self.state
+
+class GCEFloatingIP(BaseFloatingIP):
+
+    def __init__(self, provider, floating_ip):
+        super(GCEFloatingIP, self).__init__(provider)
+        self._ip = floating_ip
+
+        # We use regional IPs to simulate floating IPs not global IPs because
+        # global IPs can be forwarded only to load balancing resources, not to
+        # a specific instance. Find out the region to which the IP belongs.
+        self._region = GCPResourceUrl.parse_url(self._ip['region']).name
+
+        # Check if the address is used by a resource.
+        self._rule = None
+        self._target_instance = None
+        if 'users' in floating_ip and len(floating_ip['users']) > 0:
+            if len(floating_ip['users']) > 1:
+                cb.log.warning('IP is in user by more than one resource')
```

Comment:
cb.log.warning('Address "%s" in use by more than one resource', floating_ip['address'])
Similar change to format "Address ..." to the two warnings below.